cellprofiler/__main__.py

```diff
@@ -766,7 +766,7 @@ def run_pipeline_headless(options, args):
         else:
             image_set_start = int(options.first_image_set)
     else:
-        image_set_start = None
+        image_set_start = 1
```

Comment:
@orgonth What's going on here?

Reply:
The method `Pipeline.run()` has a parameter `image_set_start`, which default value was changed from `None` to 1 in CP4.
However, in headless mode, `image_set_start` is always explicitly set when calling the method, based on the command-line parameter `--first-image-set`. I simply changed the value if that parameter is not given, to agree with the new default of `Pipeline.run()`.